packages/generator/src/parser/edmx-parser-common.ts

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import { PathLike } from 'fs';
+import { basename } from 'path';
+import { createLogger } from '@sap-cloud-sdk/util';
+import { forceArray } from '../generator-utils';
+import {
+  EdmxMetadataBase,
+  EdmxEntityTypeBase,
+  EdmxComplexTypeBase,
+  EdmxFunctionImportBase
+} from './parser-types-common';
+
+const logger = createLogger({
+  package: 'generator',
+  messageContext: 'edmx-parser'
+});
+
+export function parseBaseMetadata(
+  root,
+  oDataVersion: 'v2' | 'v4',
```

Comment:
Minor: I have checked and the literal type `'v2' | 'v4'` appears almost a dozen times in the code. I would prefer to define it  like ODataVersionType and use it. Makes find and replace easier.

Reply:
done